### PR TITLE
docs: update samples README.md

### PR DESCRIPTION
**Pull request by stephaniewang526**

### CODE REVIEW

File: samples/README.md [hunk 1]
```diff
@@ -2,16 +2,26 @@
 
 ## Running a sample using Cloud Shell
 
-The Google Cloud Shell has application default credentials from its compute instance which will allow you to run an integration test without having to obtain `GOOGLE_APPLICATION_CREDENTIANS`.
+The Google Cloud Shell has application default credentials from its compute instance which will allow you to run an integration test without having to obtain `GOOGLE_APPLICATION_CREDENTIANS`. Go to [BigQuery Client Readme](https://github.com/googleapis/java-bigquery#samples) to run each sample in the Cloud Shell.
```
Comment (lesv) [inline samples/README.md:5]: GOOGLE_APPLICATION_CREDENTIALS

File: samples/README.md [hunk 2]
```diff
@@ -2,16 +2,26 @@
...
 
-However, certain samples require environment variables to be set. For instance, in `CreateDatasetIT.java`:
+## Running a sample using command line
+
+First set up `GOOGLE_APPLICATION_CREDENTIANS` and `GOOGLE_CLOUD_PROJECT` environment variables before running any samples.
```
Comment (lesv) [inline samples/README.md:9]: `GOOGLE_APPLICATION_CREDENTIANS` ???

File: samples/snippets/src/main/java/com/example/bigquery/SimpleQuery.java [hunk 3]
```diff
@@ -26,9 +26,15 @@
...
+    if (args.length == 0) {
+      // no command-line args passed
+      // TODO(developer): Replace this query before running the sample.
+      String query = "SELECT corpus FROM bigquery-public-data.samples.shakespeare GROUP BY corpus;";
+      simpleQuery(query);
+    } else {
+      // use command-line args
+      simpleQuery(args[0]);
+    }
   }
```
Comment (lesv) [inline samples/snippets/src/main/java/com/example/bigquery/SimpleQuery.java:38]: I don't believe this matches the style guide.

File: samples/README.md (hunk 1 above)
Comment (stephaniewang526) [reply samples/README.md]: done

File: samples/README.md (hunk 2 above)
Comment (stephaniewang526) [reply samples/README.md]: done

**Pull request merged**